TI-reports/2025/2025-Q2-BEST-WG.md

```diff
@@ -0,0 +1,224 @@
+* Continued revision, updates, & enhancement, e.g., keeping the compiler options hardening guide up-to-date with upstream options additions and changes in GCC and Clang/LLVM and addressing feedback from Linux distribution communities.  
+* Resumed work on [Annotation Guide](https://github.com/ossf/wg-best-practices-os-developers/blob/main/docs/Compiler-Hardening-Guides/Compiler-Annotations-for-C-and-C%2B%2B.md) which provides guidance on leveraging attributes to annotate C/C++ code with the goal of enabling a compiler to perform better code analysis.
+
+### Up next
+
+* Microsoft MSVC guidance planned for 2025 (tracked in [BEST Issue 150](https://github.com/ossf/wg-best-practices-os-developers/issues/150))  
+* Plan outreach activities for 2025, possibly talks aimed at C++ conferences.
+
+## Python Secure Coding Guide
+
+### Purpose
+
+* Help Python developers to create more secure code by explaining vulnerable and non-vulnerable coding patterns based on the CWE framework and rules.  
+* Besides a description of each coding pattern, the guide includes executable code examples for each rule, which allow for an in-depth understanding of each pattern.
+
+### Current Status
+
+* The group is working on adding more content for a broad range of CWE rules. The status is being tracked in issue [531](https://github.com/ossf/wg-best-practices-os-developers/issues/531).  
+* Feedback from and discussion with Seth Larson on the content and community outreach.
+
+### Up Next
+
+* We invited people from MITRE for feedback and further collaboration. In particular, we are looking for feedback regarding our use of CWEs to structure the guide.
+
+## Memory Safety SIG
+
+### Purpose
+
+* The Memory Safety SIG is a group working within the OpenSSF's Best Practices Working Group formed to understand and reduce memory safety vulnerabilities in OSS.
+
+### Current Status
+
+* Released the [Memory Safety Continuum](https://github.com/ossf/Memory-Safety/blob/main/docs/memory-safety-continuum/memory-safety-continuum.md) document
+* Scorecard release of memory safety check (unsafe block check)
+
+### Up Next
+
+* Focus on interoperability and interfacing between memory safe by default and non-memory safe by default languages in software.
+
+## OpenSSF Scorecard
+
+### Purpose
+
+* Automate analysis and trust decisions on the security posture of open source projects.  
+* Use this data to proactively improve the security posture of the critical projects the world depends on.
+
+### Current Status
+
+* Progress on implementing and testing experimental probes (e.g., memory safety).  
+* Discussion on personas and roles of maintainers and consumers in relation to Scorecard.  
+* Consideration of including sponsorship in determining maintenance level of a project.
+
+### Up Next
+
+* Make OpenSSF Scorecard (and its GitHub Action) easier to run on groups of repositories  
+* Build contributor base  
+* Work towards project graduation  
+* Request TI funding for security audit  
```

Comment:
This was done, I think? https://github.com/ossf/tac/issues/449

Reply:
I removed this outdated reference.